### PR TITLE
Updates to builder-00; IB cards

### PR DESCRIPTION
**Pull request by shall1024**
builder-00 has all of the correct cards now, and the card we were using from node-08 has been replaced.

All IB cards that we could install into the nodes have been installed and documented. We have cabled almost all of the cards as well; we are currently waiting for cable(s) for the Mellanox CX654106A cards at this time.

### CODE REVIEW

File: ib_fabric.md [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+that port is connected to a breakout cable, either to 10G or 25G, the physical
+sub-port is indicated as the second number.  *Logical ports will be updated
+when available.*
 
 ## Test Nodes
 
-TBD
+The following test nodes have **Mellanox MCX653106A-HDAT 200G 2-port** cards installed in PCI Slot 3:
+
+* node-07.ofa.iol.unh.edu - Connected to switch port 1 *and Ethernet switch port 8*
```
Comment (lylavoie) [inline ib_fabric.md:18]: Can we reference the specific switches names / models, so it's more clear what is connected to what.

File: ethernet_fabric.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ hosted in the lab.
 
 ## Switch & Fabric
```
Comment (lylavoie) [inline ethernet_fabric.md:6]: Can we also document the switch management host name and IPv4 address in this section as well. Don't publish the management password, but this just makes it easier to know how to reach the switch.

File: ib_fabric.md [hunk 3]
```diff
@@ -1,11 +1,28 @@
-# Infaband Fabric
+# InfiniBand Fabric
 
-The following information is for the Infaband (IB) Fabrics hosted within the lab.
+The following information is for the InfiniBand (IB) Fabrics hosted within the lab.
 
 ## Switch & Fabric
```
Comment (lylavoie) [inline ib_fabric.md:5]: Can we also document the switch management host name and IPv4 address in this section as well. Don't publish the management password, but this just makes it easier to know how to reach the switch.

**Comment by lylavoie** [comment]
@shall1024  can you add the connection info for builder as well, for port connecting to the Ethernet and IB fabrics.

**Comment by dledford** [comment]
I'm merging this.  If there are still updates needed, we can make a new pull request when they are available.  That way this is as up to date as possible.

**Pull request merged**